required/graphics/graphicx.dtx

```diff
@@ -170,7 +170,10 @@
 % \item[page] The page of a multi-page PDF graphic to be used.
 % \item[interpolate] Enables interpolation of bitmap images by viewers.
 % \item[pagebox] Specifies which PDF box should be used for the natural image size, one of
-% mediabox, cropbox, bleedbox, trimbox, artbox. The default is driver-specific.
+%   mediabox, cropbox, bleedbox, trimbox, artbox. The default is driver-specific.
+% \item[alt] Alternative text in accessibility uses.
+% \item[actualtext] Alternative text in accessibility uses.
```

Comment:
Should this say ActualText instead of Alternative text?

Reply:
er, yes will fix thanks